lib/hets_instance_worker.rb

```diff
@@ -0,0 +1,14 @@
+require 'sidekiq/worker'
+
+class HetsInstanceWorker < BaseWorker
+  def self.schedule_update(hets_instance_id)
+    update_span = Settings.hets.time_between_updates.hours
+    perform_in(update_span, hets_instance_id)
```

Comment:
I think we should specify a queue to ensure that it doesn't get into the unnamed queue.